vignettes/r-developer-guide.Rmd

```diff
@@ -24,129 +24,178 @@ knitr::read_chunk(path = system.file("plumber", "loess", "plumber.R", package =
                   labels = "loess")
 ```
 
-### Setup
-Install the R package.
+## Getting Started {#getting-started}
+
+This vignette will walk you through writing plumbertableau extensions in R. plumbertableau extensions are Plumber web services with a few additional pieces, and this vignette assumes some familiarity with Plumber (https://www.rplumber.io/). Plumber's [Quickstart guide](https://www.rplumber.io/articles/quickstart.html) gives a good overview of that package. Throughout this guide we'll use the terms "web service" and "API" (application programming interface) interchangeably.
```

Comment:
I think most R users familiar with Plumber will be more familiar with the term API vs. web service

```suggestion
This vignette will walk you through writing plumbertableau extensions in R. plumbertableau extensions are Plumber APIs with a few additional pieces, and this vignette assumes some familiarity with Plumber (https://www.rplumber.io/). If you're unfamiliar with Plumber, the [Quickstart guide](https://www.rplumber.io/articles/quickstart.html) gives a good overview of that package. Throughout this guide, we'll use the terms "web service" and "API" (application programming interface) interchangeably.
```